pavelib/utils/test/utils.py

```diff
@@ -17,7 +17,7 @@ def clean_test_files():
     """
     Clean fixture files used by tests and .pyc files
     """
-    sh("git clean -fqdx test_root/logs test_root/data test_root/staticfiles test_root/uploads")
+    sh("git clean -fqdx test_root/logs test_root/data test_root/uploads")
```

Comment:
@clytwynec, you are exactly right. Before my changes, Bok Choy was run with ``DEBUG=True`` which would mean that the static assets were found directly from the source tree and the generated assets under test_root were ignored. This meant that it didn't matter that the files were deleted every time. With my changes, the tests now run using the files from test_root, and so it made it impossible to use fasttest.

Can we just go with the change as I have it and create a PR to have better support for fasttest? There already appears to be unnecessary work done during fasttest, e.g. database migrations.